app/serializers/api/product_serializer.rb

```diff
@@ -49,10 +49,17 @@ class Api::CachedProductSerializer < ActiveModel::Serializer
   has_many :images, serializer: Api::ImageSerializer
   has_one :supplier, serializer: Api::IdSerializer
 
+  #return an unformatted descripton
   def description
     strip_tags object.description
   end
 
+  #return a sanitized html description
+  def description_html
+    d = sanitize(object.description, options = {tags: "p, b, strong, em, i"})
```

Comment:
Ok, awesome. Thanks for that explanation. Totally agree that we need sanitisation post-DB, no arguments there.

My understanding is that the only pre-save sanitisation happening is coming from textAngular, there is no server-side sanitisation whatsoever. This means that a malicious user could theoretically construct a POST request with an unsanitised product description which would be saved straight to the DB. If we had a pre-save callback on the model that sanitised the description on save, that would give me more peace of mind. What do you think?

What I am particularly concerned about is another developer who has not been a part of this conversation taking either allowing a user to set **description_html** value for a product without textAngular (meaning it may not be sanitised), or taking the value in the DB, assuming it is sanitised and calling html_safe on it, when in fact as it currently stands we cannot be sure of that...